neofs/neofs_contract.go

```diff
@@ -520,6 +531,17 @@ func innerRingInvoker(ir []node) []byte {
 	return nil
 }
 
+func committeeInvoker(cmt [][]byte) []byte {
+	for i := 0; i < len(cmt); i++ {
+		node := cmt[i]
+		if runtime.CheckWitness(node) {
```

Comment:
This means that each committee member has to invoke `innerRingUpdate` and I'm not sure this is the way it's supposed to work. At the same time you probably can't really check for committee multisignature.